assets/startup.sh

```diff
@@ -38,29 +38,30 @@ if [ -z "${LOG_LEVEL}" ]; then
   LOG_LEVEL=0
 else
   # confirm log level is between 0-3, since these are the only log levels supported
-  if [ "${LOG_LEVEL}" -gt 3 ]; then
+  if [[ $LOG_LEVEL != [0123] ]]; then
     # level outside of supported range, let's set to default (0)
     LOG_LEVEL=0
   fi
 fi
 
-IFS=","
+IFS=", "
+set -f
```

Comment:
This turns off globbing (in bash set). Apart from doing this as a precaution, is there a concrete reason to do so here? I would either remove it if there it is only added as a precaution, or if there is a need, then move it to the start of the script, where I would expect to find set restrictions.

If the need is only around a certain section, do place it close to the needed area and then undue the side-effect again with a `set +f` as soon as possible.